event-app-api/merge_intervals/merge_intervals.js

```diff
@@ -0,0 +1,42 @@
+export function mergeIntervals(testEvents3) {
```

Comment:
testEvents3 should just be named testEvents

Reply:
I will change that now